source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/input/customcontrols/ValidatedCheckBox.java

```diff
@@ -0,0 +1,72 @@
+package io.adaptivecards.renderer.input.customcontrols;
+
+import android.content.Context;
+import android.graphics.Color;
+import android.graphics.PorterDuff;
+import android.support.v4.content.ContextCompat;
+import android.widget.CheckBox;
+import android.widget.LinearLayout;
+
+import io.adaptivecards.R;
```

Comment:
>.R; [](start = 23, length = 3)

This is a very interesting namespace! What gives?

Reply:
R is the name of the container that includes drawables, color definitions, attribute definitions and other resources

https://developer.android.com/reference/android/R